torchao/_models/mixtral-moe/model.py

```diff
@@ -310,7 +310,7 @@ def apply_rotary_emb(x: Tensor, freqs_cis: Tensor) -> Tensor:
 # T'(e) tokens for expert e
 
 
-class MOEFeedForwardAOQuantizable(nn.Module):
+class MoEFeedForwardAOQuantizable(nn.Module):
```

Comment:
It seems unlikely that people are going to swap their MoE module to AO's version.  Can we just target `torch._grouped_mm` calls directly without requiring a module swap?

Reply:
What would it mean to "target" it specifically?  If given model compiled, the compiled version of this operator will be used anyway, not sure what else torchao could do about it...